packaging/datadog-agent/source/install_agent.sh

```diff
@@ -40,6 +40,12 @@ if [ -n "$DD_API_KEY" ]; then
     apikey=$DD_API_KEY
 fi
 
+if [ -n "$DD_NO_START" ]; then
```

Comment:
On second thought, you should probably call it `$DD_NO_RESTART`, because it's really a restart. But when set to true during an upgrade, it will restart anyway.
Even with `$DD_NO_RESTART`, I'm afraid that people may think that it actually always prevents the agent from restarting, even during upgrades...
`DD_NO_FIRST_RESTART` maybe ? That sounds ridiculous, do you have a better idea ?